api/libproject/devstart.rb

```diff
@@ -73,6 +73,14 @@ def run_api(account)
   end
 end
 
+def run_public_api_and_db()
+  common = Common.new
+  common.status "Starting database..."
+  common.run_inline %W{docker-compose up -d db}
```

Comment:
Do you need to explicitly start the db service since the public-api service depends on it?